plugins/node/instrumentation-runtime-node/package.json

```diff
@@ -20,7 +20,7 @@
   "author": "OpenTelemetry Authors",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=14.10.0"
+    "node": ">=17.4.0"
```

Comment:
I think this should be considered as a breaking change.

May I ask which APIs require this version change? Do you think is possible to send metrics available for versions `<17.4.0`?

Reply:
https://nodejs.org/api/perf_hooks.html#histogramcount
I need to use histogram.count that was added in 17.4